tests/plugins/test_reactor_config.py

```diff
@@ -114,6 +121,7 @@ def get_resource_stream(self, pkg, rsc):
         captured_errs = [x.message for x in caplog.records()]
         assert any("cannot validate" in x for x in captured_errs)
 
+    @pytest.mark.parametrize('reactor_config_map', [True, False])
```

Comment:
Maybe we could make this a fixture, to avoid duplicating code in the tests?

Reply:
ok will do